Bottom: 6e5265b31786f8d5a7dc52446a3057486c6f770d
Top:    26947f4c017d0005c80338bceaf0f2fc66d784dd
Author: Ian Kent <ikent@redhat.com>
Date:   2013-09-13 16:39:02 +0800

Refresh of autofs-5.0.7-teach-dumpmaps-to-output-simple-key-value-pairs.patch

---

diff --git a/lib/master.c b/lib/master.c
index 964b547..ce6320e 100644
--- a/lib/master.c
+++ b/lib/master.c
@@ -1339,7 +1339,7 @@ static int match_type(const char *source, const char *type)
 	return 0;
 }
 
-static char *get_map_name(char *string)
+static char *get_map_name(const char *string)
 {
 	char *name, *tmp;
 	char *start, *end, *base;
